src/main/pg/board.c

```diff
@@ -38,8 +38,10 @@ PG_REGISTER_WITH_RESET_FN(boardConfig_t, boardConfig, PG_BOARD_CONFIG, 0);
 void pgResetFn_boardConfig(boardConfig_t *boardConfig)
 {
     if (boardInformationIsSet()) {
-        strncpy(boardConfig->manufacturerId, getManufacturerId(), MAX_MANUFACTURER_ID_LENGTH + 1);
-        strncpy(boardConfig->boardName, getBoardName(), MAX_BOARD_NAME_LENGTH + 1);
+        strncpy(boardConfig->manufacturerId, getManufacturerId(), sizeof(boardConfig->manufacturerId));
```

Comment:
(sorry, I overlooked this in my suggestion)
```suggestion
        strncpy(boardConfig->manufacturerId, getManufacturerId(), sizeof(boardConfig->manufacturerId) - 1);
```